assets/src/blocks/Covers/getCoversClassName.js

```diff
@@ -0,0 +1,19 @@
+import { COVER_TYPES } from './Covers';
+
+export const getCoversClassName = (cover_type, initialRowsLimit) => {
+  const isContentType = cover_type === COVER_TYPES.content;
+  const isTakeActionType = cover_type === COVER_TYPES.takeAction;
+
+  let rowClassName = isContentType ? 'show-all-rows' : 'show-all-covers';
```

Comment:
Doesn't this PR also delete these classes? https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/486/files#diff-5d4325cb4bf033f21bfb7aa6b87ecb070d875d33c687d00d8cb70e71d7400968L55

Reply:
Removed!